youtube_to_anki/youtube/download.py

```diff
@@ -35,3 +47,12 @@ def retrieve_audio(url: str) -> AudioSegment:
     with TemporaryDirectory() as tempdir:
         _download_audio(url, f"{tempdir}/temp.%(ext)s")
         return AudioSegment.from_mp3(f"{tempdir}/temp.mp3")
+
+def retrieve_video(url: str, resolution: int) -> cv2.VideoCapture:
+    """
+    Downloads audio from YouTube URL and returns it as AudioSegment.
+    """
+    with TemporaryDirectory() as tempdir:
+        path = f"{tempdir}/temp.mp4"
```

Comment:
it's not always `.mp4`, but the whole file extension login in the youtube downloader seems like a pain to deal with, in my testing so far the extraction always worked for the `.mp4` extension.

on a sidenote: I get an error when trying to run `youtube-to-anki pq_4FTG4VAE --transcript-language ja`, seemingly caused by the audio downloader expecting an `mp3`, but actually getting a `wemb`